src/orquestra/vqa/cost_function/cost_function.py

```diff
@@ -125,7 +125,7 @@ def _preprocess(parameters: np.ndarray) -> np.ndarray:
 
 
 def create_cost_function(
-    backend: QuantumBackend,
+    backend: BaseCircuitRunner,
```

Comment:
`BaseCircuitRunner` is not meant for typing. I think that the correct type hint here is just `CircuitRunner`. Also, I think that we should name arguments accepting `CircuitRunner` as `runner` not `backend`.

Reply:
Done